content/resources/registry/publishing/_index.md

```diff
@@ -0,0 +1,183 @@
+We recommend copying the Markdown text and adding it to your README-file in your GitHub Recocnizer Repository.
+
+#### Step 7. Add or modify a citation information file
+
+To make your GitHub recognizer repository easily citable we recommend adding a citation information file (`CITATION.cff`).  We have added a citation file to the recognizer template for your convenience. 
+
+1. Modify the `CITATION.cff` file in your repository.
+![Screenshot of the Citation.cff file.](citation-file-edit.png)
+You can edit files in your browser by clicking the pen icon.
+Replace the author name, the ORCID, the title and the github link fields with your own details.
+2. Alternatively you can use the
+[CITATION.cff generation website](https://citation-file-format.github.io/cff-initializer-javascript/#/) to create the
+`CITATION.cff` file  
+3. Once you are happy with your changes, save the file by clicking
+the "Commit changes..." button. You can commit your changes directly to the main branch. 
+4. If you will be collaborating with multiple authors we recommend [learning more about working with branches in GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches). 
+
+#### Step 8. List your recognizer in the registry
+Please see the [Contribute to the registry](/resources/registry/contribute/) page. 
+
+## FAQ
+
+### What to do when your training data set is too large?
+
+Storing data in a repository is not always the right choice. In each folder where it is relevant you should include:
+
+1. Small sets of audio samples 
+2. A README.md containing
+  - provenance of any data included
+  - instructions on how to obatain more data
+3. Any scripts needed to download data from remote repositories
+
+__For larger datasets you can use:__
+
+- An ecoacoustics repository
+    - like Ecosounds, the A2O, or ...others...
+- A bioacoustics repository
+    - like Xeno Canto
+- Cloud storage options like DropBox, CloudStor, OneDrive, etc.
+- Commersial services like Amazon S3, Google Cloud Storage, etc.
+
+### How do I keep my recognizer private until I'm ready to publish?
+If you wish to keep your recognizer under a private embargo, you have the option to set the visibility of your repository when you create it, or later from the repository settings.
+
+#### How to change the repository settings?
+1. In your repository go to the Settings tab
+2. In the _General_ section, scroll to the bottom "Danger Zone"
+3. Click the button _Change visibility_ and select  _Change to private_. Once you are ready to publish, go to this same setting and select _Change to public._
+![Screenshot of Visibility settings](github-change-visibility.png)
+
+See more in GitHub Documentation:
+[setting repository visibility](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/managing-repository-settings/setting-repository-visibility).
+
```

Comment:
Other FAQ's:

- Who owns my recognizer? You! That's one of the reasons we think GitHub is great: you control and own your IP and this guide is just helping you do that
- Is the template and GitHub still a good choice for a recognizer that will only ever be private? Yes! version control, access control, common pattern for publishing, all other advantages listed